public/app/features/alerting/unified/components/contact-points/__snapshots__/useContactPoints.test.tsx.snap

```diff
@@ -263,13 +269,16 @@ exports[`useContactPoints when having oncall plugin installed and no alert manag
             "name": "Grafana OnCall",
           },
           Symbol(receiver_plugin_metadata): {
+            "description": "grafana-integration",
```

Comment:
I believe this is because the test waits for the loading states to all settle slightly differently now - so the snapshot is more accurate now

Reply:
Nevermind, it was because I didn't disable the on call plugin in the tests now :D